src/post/Write/DeleteDraftModal.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Modal } from 'antd';
+import 'url-search-params-polyfill';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import { deleteDraft } from './editorActions';
+import { notify } from '../../app/Notification/notificationActions';
+
+@injectIntl
+@connect(null, {
+  notify,
+  deleteDraft,
+})
+class DeleteDraftModal extends React.Component {
+  static propTypes = {
```

Comment:
Can you sort props that functions are at the bottom? In propTypes and defaultProps. We tend to use this code styles everywhere.